internal/manifests/distributor/distributor.go

```diff
@@ -338,3 +357,30 @@ func service(tempo v1alpha1.TempoStack) *corev1.Service {
 		},
 	}
 }
+
+func getTLSSecretName(tempoName string) string {
+	return fmt.Sprintf("%s-distributor-tls", tempoName)
+}
+func getTLSConfigMapName(tempoName string) string {
+	return fmt.Sprintf("%s-distributor-ca", tempoName)
+}
+
+func getConfigmapCABundle(tempo v1alpha1.TempoStack) *corev1.ConfigMap {
+	name := getTLSConfigMapName(tempo.Name)
+	annotations := map[string]string{
+		"service.beta.openshift.io/inject-cabundle": "true",
+	}
+
+	return &corev1.ConfigMap{
```

Comment:
We use this in a few places, you can use `tls.NewConfigMapCABundle()` to generate this configmap.

Reply:
Done